### ISSUE TITLE
Workshop on making the most of hack night?  Explain the structure/schedule

### ISSUE THREAD

**Issue by Aycrazy** [codeformilwaukee/hack-night-planning#116]
I'm thinking that this could replace a presentation one month.  Will need everyone's input on this, could be the plan for February Hack Night

@mitchellhenke @romkedehaan @erinmagennis @BarkleyBG @anthonyjesmok @esquivelan01

**Comment by BarkleyBG** [comment]
It’s not totally clear to me what this means  @aycrazy. Is it that, instead of a presentation, we could provide a presentation to the audience on what our Hack Nights are about, and what we hope the audience will get out of it?

**Comment by Aycrazy** [comment]
Yeah and maybe best practices for making the most of one's time (if we assume they want to find a project to work on and to move that project forward in some way). I don't know if that would be much of a draw though. So maybe presenting on this should be in a different setting
